hack/make-rules/vet.sh

```diff
@@ -25,8 +25,14 @@ cd "${KUBE_ROOT}"
 
 # This is required before we run govet for the results to be correct.
 # See https://github.com/golang/go/issues/16086 for details.
-make generated_files
-go install ./cmd/...
```

Comment:
ISTR the `go install` bit was important too, though I'm not sure if it still is. this is my main concern with this PR.